docs/rules/no-webpack-loader-syntax.md

```diff
@@ -0,0 +1,36 @@
+# no-webpack-loader-syntax
+
+Forbid Webpack loader syntax in imports.
+
+[Webpack](http://webpack.github.io) allows specifying [loaders](http://webpack.github.io/docs/loaders.html) and their configuration inline in imports using a special syntax like this:
```

Comment:
Suggestion:
`Webpack allows specifying the loaders to use in the imports source string using ...`